src/kernel/arch/x86/paging.zig

```diff
@@ -372,7 +372,8 @@ test "virtToPhys" {
     const offset: usize = @ptrToInt(&KERNEL_ADDR_OFFSET);
     expectEqual(virtToPhys(offset + 0), 0);
     expectEqual(virtToPhys(offset + 123), 123);
-    expectEqual(@ptrToInt(virtToPhys(@intToPtr(*usize, offset + 123))), 123);
+    if (builtin.mode != builtin.Mode.ReleaseSafe)
```

Comment:
XD, This is cheating. I think it is bc ur comparing a pointer to a uint. `@ptrToInt() != u32`

Reply:
:( I hoped you wouldn't notice the sneakyness. How do you think I could fix this?

Reply:
I've added a workaround. It's not nice, but is all I can get working at the moment. I'll add an issue to monitor it once this is merged.